transport/grpc/peer.go

```diff
@@ -35,12 +33,11 @@ type grpcPeer struct {
 	*hostport.Peer
 
 	t          *Transport
+	ctx        context.Context
+	cancel     context.CancelFunc
+	status     peer.ConnectionStatus
```

Comment:
This is duplicative since we can lift this directly from the embedded `*hostport.Peer`, right?

Reply:
Never mind, I understand now. Can we leave a comment?